common/settings.h

```diff
@@ -207,6 +207,12 @@ struct Settings {
   /// https://github.com/dart-lang/sdk/blob/ca64509108b3e7219c50d6c52877c85ab6a35ff2/runtime/vm/flag_list.h#L150
   int64_t old_gen_heap_size = -1;
 
+  /// A timestamp representing when the engine started. The value is in
+  /// microseconds based on the clock used by the Dart timeline APIs. This
+  /// timestamp is used to log a timeline event that tracks the latency of
+  /// engine startup.
+  int64_t engine_start_timestamp = 0;
```

Comment:
Lets use std::chrono instead of depending on the int representing the timestamp in milliseconds.

Reply:
done